src/Analyzers/CSharp/Analyzers/UseConditionalExpression/CSharpUseConditionalExpressionHelpers.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable enable
+
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Operations;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.CSharp.UseConditionalExpression
```

Comment:
Can this file be just in the shared `CodeFixes.shproj` instead of `Analyzers.shproj`? Seems to have code fix specific logic.

Reply:
sure.